containers/bash-scheduled-job/app/server.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+### Create the response FIFO
```

Comment:
Can you add a little explanation at the top of this script to say what it's doing? What's the FIFO used for, and what happens with different HTTP methods?